src/utils/config.ts

```diff
@@ -329,11 +329,14 @@ export const RECAPTCHA_SCORE_THRESHOLD = 0.5;
 export const GET_MOST_LIKED_ITEMS_MAXIMUM = 50;
 export const GET_MOST_RECENT_ITEMS_MAXIMUM = 50;
 
+// user feedback reporting
+export const USER_FEEDBACK_PREFIX = '/user-feedback';
+export const USER_FEEDBACK_RECEIVER_EMAIL = process.env.USER_FEEDBACK_RECEIVER_EMAIL;
```

Comment:
For this environnement variable, I would use the same pattern as for the `RECAPTCHA_SECRET_ACCESS_KEY`. The app should exit if it not set. I think it makes sens to require it, as otherwise the feature is not able to work as expected.